### PR TITLE
[CBRD-23732] Optional Statistics Estimation

### PR DESCRIPTION
**Pull request by shparkcubrid**
http://jira.cubrid.org/browse/CBRD-23732

it is added system parameter 'use_stat_estimation' to prevent automatic update of statistics.
if use_stat_estimation is no then following behaviors is changed.
```
do not add estimated rows to statistics
```

The following are not fixed in this issue. the sampling update statistics is performed like as-is.
```
update statistics when altering table
```

### CODE REVIEW

**Comment by cubridci** [comment]
Build comment file: 
--------------
**
** This Pull Request's title is 'Optional Statistics Estimation'.
** And it does not start with a related JIRA issue number **
** Please edit this Pull Request's title. (eg: '[CBRD-20309] Redesign vacuum data flush' **
**

--------------

**Comment by shparkcubrid** [comment]
CTC errors will be checked again when the test case is corrected normally.

**Review by byungwook-kim — APPROVED** [review]

**Review by hgryoo — APPROVED** [review]

File: src/object/schema_manager.c [hunk 1]
```diff
@@ -14785,7 +14786,16 @@ sm_add_constraint (MOP classop, DB_CONSTRAINT_TYPE constraint_type, const char *
 	      goto error_exit;
 	    }
 	}
-
+      else if (!use_stat_estimation)
+	{
+	  /* if NOT use_stat_estimation then stats is not updated at update_class() */
+	  /* this routine is for create index statement */
+	  error = sm_update_statistics (newmop, STATS_WITH_SAMPLING);
```
Comment (byungwook-kim) [inline src/object/schema_manager.c:14793]: if the statistics are changed just only when an index is created, what happen for updating, inserting, or deleting?

Comment (shparkcubrid) [reply src/object/schema_manager.c:14793]: use_stat_estimation=no
statistics are not changed when updating, inserting, deleting.
Only when executing 'update statistics' stmt, 'create index' stmt and 'drop index' stmt, the statistics are changed.

**Review by hgryoo — APPROVED** [review]

**Pull request merged**